### DISCUSSION TITLE
建议为图片加载提供懒加载或并发限制

### DISCUSSION THREAD

**Discussion by Arc-2023** [imzbf/md-editor-v3#293] [Ideas]
![image](https://github.com/imzbf/md-editor-v3/assets/64178177/a7076e59-c0c9-45bd-827e-d3b35928540f)
如上，当md文档图片数量过多时，大量<img/>标签并发请求后端
在服务器带宽低的情况下会导致图片无法加载、带宽大量占用且无法释放等问题：
如后端minio报错：
![image](https://github.com/imzbf/md-editor-v3/assets/64178177/33af50e1-ac08-4ebe-885c-3221bdc2d6d7)

**Comment by imzbf** [comment]
懒加载这类需求需要开发者自行实现，[参考](https://github.com/imzbf/md-editor-v3/issues/279#issuecomment-1569438463)来扩展`markdown-it`的图片模块

**Comment by Arc-2023** [comment]
已通过caddy解决此问题
但是效果不如后端和前端本身对并发的限制
具体实现：
```
handle /getImage/* {
    reverse_proxy http://ip:port {
      transport http {
       max_conns_per_host 3
     }
   }
  }
```
